README.md

```diff
@@ -2,6 +2,7 @@
 
 A serverless form submission application built and deployed to AWS with the Serverless Application Framework.
 
+
 ## Architecture
 
 ![Architecture Diagram](./.images/architecture.png?raw=true)
```

Comment:
its hard to leaving a meaningful comment in a PR like this but I PM'ed @ayoneeyee and asked him to remove the cognito portion of the diagram that was leftover from elasticsearch